functions/venue.js

```diff
@@ -636,17 +636,17 @@ exports.updateVenue = functions.https.onCall(async (data, context) => {
 
   // @debt this is missing from updateVenue_v2, why is that? Do we need it there/here?
   //   I expect this may be legacy functionality related to the Playa template?
-  if (
-    !data.placement.state ||
-    data.placement.state === PlacementState.SelfPlaced
-  ) {
-    updated.placement = {
-      ...data.placement,
-      state: PlacementState.SelfPlaced,
-    };
-  } else if (data.placementRequests) {
-    updated.placementRequests = data.placementRequests;
-  }
+  // if (
+  //   !data.placement.state ||
+  //   data.placement.state === PlacementState.SelfPlaced
+  // ) {
+  //   updated.placement = {
+  //     ...data.placement,
+  //     state: PlacementState.SelfPlaced,
+  //   };
+  // } else if (data.placementRequests) {
+  //   updated.placementRequests = data.placementRequests;
+  // }
```

Comment:
Why was this commented out?

Reply:
This is playa legacy code and it blows up for some of the venues that were created from the admin v3

Reply:
If we haven't already can we please `@debt` it to explain that.

Reply:
@0xdevalias There is debt about it, explaining that this is playa legacy code. We should remove these in the near future.